apidoc/Titanium/UI/WebView.yml

```diff
@@ -388,6 +388,16 @@ events:
       - name: code
         summary: SSL error code.
         type: Number
+
+  - name: onStopBlacklistedUrl
```

Comment:
Don't like the `on` prefix for events. On Alloy, it produces something like `<WebView onOnstopblacklistedurl="" />`. Can we migrate this and `onLoadResource ` to `stopblacklistedurl` and `loadresource` ?